com.cinecred.cinecred.yml

```diff
@@ -0,0 +1,41 @@
+id: com.cinecred.cinecred
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+command: cinecred
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --device=dri
+  - --filesystem=host
```

Comment:
this is a very broad filesystem permission.

Reply:
The full file system access is required because users can drag-and-drop folders into the welcome window to open them, and portals don't support that:

![Drag-and-drop demo](https://cinecred.com/guide/project-folder/create-project.gif)

Reply:
the filetransfer portal work with folders.

Reply:
Oh, I didn't know about that yet. Thanks a lot for letting me know! I'll try adding support for that portal in the next release.

Still, there are places in the UI where no portal solution would work. For example, in the delivery window, the user can select the output file either with a portal file chooser, or by pasting a file path into a text field. In addition, when the user switches the render format, the file extension is automatically updated. In the latter two cases, I think it's impossible to obtain writing permission for the file via portals.

![Delivery window](https://cinecred.com/home/screenshot-delivery-options.png)